templates/standalone-ha.xml.j2

```diff
@@ -615,8 +615,8 @@
             <scheduled-task-interval>900</scheduled-task-interval>
             <theme>
                 <staticMaxAge>2592000</staticMaxAge>
-                <cacheThemes>true</cacheThemes>
-                <cacheTemplates>true</cacheTemplates>
+                <cacheThemes>{{ keycloak_themes_cache_enabled | default('true') }}</cacheThemes>
```

Comment:
I would also add this new vars to the defaults vars, at least in a comment, so people using this role can see at a glance all variables provided by the role :)